components/crud-web-apps/tensorboards/backend/app/utils.py

```diff
@@ -5,16 +5,34 @@ def parse_tensorboard(tensorboard):
     """
     Process the Tensorboard object and format it as the UI expects it.
     """
+    
+    if tensorboard.get("status", {}).get("readyReplicas", 0) == 1: 
+        phase = "ready"
```

Comment:
Could you use the [class attributes](https://github.com/kubeflow/kubeflow/blob/master/components/crud-web-apps/common/backend/kubeflow/kubeflow/crud_backend/status.py#L1) instead of string values here?
It should be `STATUS_PHASE.READY`